File: plugin/Weather/src/main/java/weather/WeatherResponse.java
package weather;

import java.util.List;

public class WeatherResponse {
    public List<Condition> current_condition;
    public List<NearestArea> nearest_area;

    public static class Condition {
        public String FeelsLikeC;
        public String FeelsLikeF;
        public String cloudcover;
        public String humidity;
        public String temp_C;
        public String temp_F;
        public String uvIndex;
        public List<TextValue> weatherDesc;
        public String winddir16Point;
        public String windspeedKmph;
        public String windspeedMiles;
    }

    public static class NearestArea {
        public List<TextValue> areaName;
        public List<TextValue> country;
        public List<TextValue> region;
    }

    public static class TextValue {
        public String value;
    }
}

File: Weather/src/main/kotlin/weather/WeatherResponse.kt
package weather

data class WeatherResponse(
    val current_condition: List<Condition>,
    val nearest_area: List<NearestArea>
) {
    data class Condition(
        val FeelsLikeC: String,
        val FeelsLikeF: String,
        val cloudcover: String,
        val humidity: String,
        val temp_C: String,
        val temp_F: String,
        val uvIndex: String,
        val weatherDesc: List<TextValue>,
        val winddir16Point: String,
        val windspeedKmph: String,
        val windspeedMiles: String
    )

    data class NearestArea(
        val areaName: List<TextValue>,
        val country: List<TextValue>,
        val region: List<TextValue>
    )

    data class TextValue(val value: String)
}